cmake/external_mkldnn.cmake

```diff
@@ -39,7 +39,7 @@ if(NGRAPH_CPU_ENABLE)
                 -DCMAKE_C_COMPILER=${CMAKE_C_COMPILER}
                 -DCMAKE_CXX_COMPILER=${CMAKE_CXX_COMPILER}
                 -DCMAKE_INSTALL_PREFIX=${EXTERNAL_PROJECTS_ROOT}/mkldnn
-            BUILD_COMMAND ${CMAKE_MAKE_PROGRAM} mkldnn
+            BUILD_COMMAND $(MAKE) mkldnn
```

Comment:
This is incorrect and will break non Makefile generators

Reply:
I'm aware that it will break non Makefile generators. Are we supporting other generators? I would love to use Ninja. But why do we have an explicit BUILD_COMMAND in the first place? We don't specify and explicit INSTALL_COMMAND and the default INSTALL_COMMAND builds the skipped targets anyway. I can go back to my original proposal that removes the BUILD_COMMAND. That will not break non Makefile generators and does not cause "warning: jobserver unavailable: using -j1. Add '+' to parent make rule." Does that sound like the right solution?

Reply:
I can't speak to "official" support etc but following CMake best practices typically results in build scripts that users can run with any generator. Ninja works right now and I try to fix issues or point out changes that will break it.

Regarding the substance of the issue, I have no comment really. I didn't introduce the BUILD_COMMAND change. Removing it entirely works for me FWIW. My goal is to just make sure my personal workflow doesn't break